drivers/i2c/i2c_mchp_xec.c

```diff
@@ -69,6 +72,43 @@ static const struct xec_speed_cfg xec_cfg_params[] = {
 	},
 };
 
+static void i2c_xec_reconfig(uint32_t ba)
+{
+	/* Assert RESET and clr others */
+	MCHP_I2C_SMB_CFG(ba) = MCHP_I2C_SMB_CFG_RESET;
+
+	k_busy_wait(20);
```

Comment:
Why 20 us?

Reply:
The wait is supposed to be for at least 1 clock period (baud). 20 us is just a safe number.